app/routes/user.py

```diff
@@ -443,6 +443,14 @@ def get_me():
     set_access_cookies(resp, auth_token)
     return resp
 
+@bp.route("/users/nearby")
```

Comment:
instead of putting this here, let's have a separate blueprint for it. and same with `review` vs `reviews`. that way blueprints always have a path prefix

Reply:
Okay, got it.